index.html

```diff
@@ -763,16 +763,22 @@ <h2>
         Security and privacy considerations
       </h2>
       <p>
-        Wake locks can cause various device components such as display or CPU
-        to operate at higher power levels than they otherwise would. This can
-        lead to undesirable effects such as faster than normal battery charge
-        depletion. This is particularly relevant to mobile devices, which may
-        not have a stationary power source readily available. Complete battery
-        depletion at an unexpected time can lead to inability of the user to
-        make or receive calls and use network services, including the emergency
-        call service. Implementations MAY ignore requests for screen wake lock
-        if, for example, the battery capacity is low, or the user has put their
-        device in a power-saving mode.
+        Screen wake locks can cause various device components - particularly
+        the display - to operate at higher power levels than they otherwise
+        would. This can lead to undesirable effects, such as preventing the
+        device from automatically locking itself and faster battery depletion.
+        Faster battery depletion is of particular concern for mobile devices,
+        which often don't have a stationary power source readily available.
+        Complete battery depletion at an unexpected time can lead to inability
+        of the user to make or receive calls and use network services,
+        including the emergency call service.
+      </p>
+      <p>
+        It is RECOMMENDED that a user agent provide some UI or indicator that
+        allows the user to know when a wake lock is active. Providing such a UI
```

Comment:
s/wake lock/screen wake lock/ ?

Reply:
It appears that we have lost this sentence "Implementations MAY ignore requests for screen wake lock
         if, for example, the battery capacity is low, or the user has put their
         device in a power-saving mode."
The fact that we have now added text to notify the user when wake lock is active, does not negate the need to say that implementations may ignore wake-lock requests. In fact, we could also Recommend that implementations ignore wake-lock requests when the battery capacity is low

Reply:
I've sent #328 to add this back.